token_values_example_draft_v2.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Hey @ruben-pulido, just for clarification... who is meant to produce and store this format? I'm assuming this is a reference for the StyleBook app and similar ones? Are you storing this JSON in the database and then consuming it to generate the `ScopedCSSVariables` and the edition UI?

As a `token_values` json makes sense to me.

I just want to point out that to generate the `ScopedCSSVariables` you'll need to read both `token_values.json` and the whole `token_definition.json` and then map from `name` to `mappings/css_variables[name]`... which does not look terribly bad, so just thinking out loud here... 🤔

Reply:
> Hey @ruben-pulido, just for clarification... who is meant to produce and store this format? I'm assuming this is a reference for the StyleBook app and similar ones? Are you storing this JSON in the database and then consuming it to generate the `ScopedCSSVariables` and the edition UI?
> 


Yes, this a reference for StyleBook app and similar ones.
This JSON will be stored in the database (for StyleBook app in the `tokenValues` column of the `StyleBookEntry` table).

> 
> I just want to point out that to generate the `ScopedCSSVariables` you'll need to read both `token_values.json` and the whole `token_definition.json` and then map from `name` to `tokens[name]/mappings/css_variables`... which does not look terribly bad, so just thinking out loud here... 🤔

Correct. To generate ScopedCSSVariables both `token_definition.json` and `token_values.json` are needed as input.

Reply:
👍